client/src/components/TopNavigationBar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import {
+  AppBar,
+  Typography,
+  Toolbar,
+  Button,
+  Menu,
+  MenuItem,
+  Divider,
+} from "@material-ui/core";
+import { ArrowDropDown } from "@material-ui/icons";
+import { makeStyles } from "@material-ui/core/styles";
+import { ReactComponent as Logo } from "images/logo.svg";
+import UserAvatar from "common/UserAvatar";
+
+const TopNavigationBar = ({ loggedInUser }) => {
+  const classes = useStyles();
+
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <AppBar style={{ background: "#fff" }} position="static">
+      <Toolbar>
+        <Logo className={classes.logo} />
+        <Button size="large">Chefs</Button>
+        <Button size="large">Dishes</Button>
```

Comment:
These buttons of "Chefs Dishes" are white text, cannot be seen by user, and should be deleted